parsers/MD.py

```diff
@@ -251,8 +228,7 @@ def fetch_production(
 ) -> list[dict] | dict:
     """Requests the production mix (in MW) of a given country."""
     if target_datetime:
-        archive_data = get_archive_data(session, target_datetime)
-
+        archive_data = get_archive_data(target_datetime, session=session)
         datapoints = []
```

Comment:
Would it be possible for you to make it only return the archive data?

I noticed the live production data also just grabs the current timestamp which is something we want to avoid if possible.

Otherwise this seems to work as expected.

Reply:
Sure! The only downside I can think of is that we would lose the more granular information about the solar / wind / biomass distribution, which seems to only be available on the live API. Happy to do that anyway?

Reply:
Perfect- will do. Should I use only `get_archive_data` in all of the other functions too?

Reply:
If possible then yes. We really want data that is mapped to an actual datetime and not just the time when the parser runs.